src/core/ui.js

```diff
@@ -59,14 +59,29 @@ respecPill.addEventListener("click", function(e) {
   this.setAttribute("aria-expanded", String(menu.hidden));
   menu.hidden = !menu.hidden;
 });
+document.documentElement.addEventListener("keyup", e => {
+  e.stopPropagation();
+  if (e.keyCode === 73) {
+    if (menu.hidden) {
```

Comment:
The following lines or code are basically identical, so why not:

```JS
const classes = ["visible", "hidden"];
const [toAdd, toRemove] =  menu.hidden ? classes.reverse() : classes;  
menu.classList.remove(`respec-${toRemove}`);
menu.classList.add(`respec-${toAdd}`);
if (menu.hidden) respecPill.focus();
```

Or, if "respec-hidden" is set at construction time, maybe even just:

```JS  
menu.classList.toggle("respec-visible");
menu.classList.toggle("respec-hidden", menu.hidden);
if (menu.hidden) respecPill.focus();
```

Reply:
Why do we have both `respec-visible` and `respec-hidden`?

Reply:
Yeah, can probably roll visible of into the base element’s css definition (or the other way around).